examples/gcp_cli.py

```diff
@@ -97,3 +98,17 @@ def QueryLogs(args: 'argparse.Namespace') -> None:
   print('Found {0:d} log entries:'.format(len(results)))
   for line in results:
     print(json.dumps(line))
+
+
+def ListServices(args: 'argparse.Namespace') -> None:
+  """List active GCP APIs for a project.
+
+  Args:
+    args (argparse.Namespace): Arguments from ArgumentParser.
+  """
+  apis = gcp_monitoring.GoogleCloudMonitoring(args.project)
+  results = apis.ActiveServices()
+  print('Found {0:d} APIs:'.format(len(results)))
+  sorted_apis = sorted(results.items(), key=lambda x: x[1], reverse=True)
+  for i in sorted_apis:
```

Comment:
What's `i`? Please use a more descriptive variable name, ideally splitting it into two explicit variables instead of using `i[0]` and `i[1]`

```python
for something, something_else in sorted_apis:
  # blah
```

Reply:
Done